MacrobenchmarkSample/app/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
     compileSdkVersion 31
     defaultConfig {
         applicationId "com.example.macrobenchmark.target"
-        minSdkVersion 14
+        minSdkVersion 21
```

Comment:
Bumped this to 21 for Compose, let me know if I need to workaround the original value

Reply:
No, `21` should be okay.